objectRecognitionSource/src/main/java/edu/ml/tensorflow/model/BoxPosition.java

```diff
@@ -10,12 +13,26 @@
     private float bottom;
     private float width;
     private float height;
-
-    public BoxPosition(float left, float top, float width, float height) {
-        this.left = left;
-        this.top = top;
+    private float centerX;
+    private float centerY;
+
+
+    /**
+     * Build from center point + height & width
+     *
+     * @param x float X coordinate of the center
+     * @param y float Y coordinate of the center
+     * @param width float width of box
+     * @param height float height of box
+     */
+    public BoxPosition(float x, float y, float width, float height) {
+
+        this.left = x - width / 2;
```

Comment:
Again, calling convention change -- we now calculate `left` and `top` here rather than have it calculated by the caller.